charts/node/values.yaml

```diff
@@ -170,6 +170,20 @@ node:
   ## If Collator is enabled, collator image must be used
   ##
   isParachain: false
+
+  ## EXPERIMENTAL!!!
+  ## Run the collator node without a relay chain
+  ## ref: https://github.com/paritytech/cumulus#external-relay-chain-node
+  ##
+  ## Enabling this option will disable the values of collatorRelayChain
+  ##
+  externalRelayChain:
+    enabled: false
+
+    ## List of Relay Chain RPCs to connect
+    relayChainRpcUrls: ["wss://rpc.polkadot.io"]
```

Comment:
I think it's better not to put a default value here (default:`[]`).

Reply:
The reason why I added the polkadot RPC is because the default chain value is set to Polkadot. I can remove it.